diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -372,7 +372,7 @@ namespace diff_drive_controller
     setOdomPubFields(root_nh, controller_nh);
 
     // Set dynamic reconfigure params defaults to the static params provided:
-    DiffDriveControllerConfig config;
+    DiffDriveControllerConfig config(config_default_);
```

Comment:
The actual problem is a few lines below.

Could you please revert this change and replace https://github.com/clearpathrobotics/ros_controllers/pull/40/files#diff-380f38b77ed7f79a1de5cf7de13f429eL392 with

```c++
config.control_frequency_desired = control_frequency_desired_;
```